packages/core/package.json

```diff
@@ -25,6 +25,7 @@
   "scripts": {
     "build": "npm run clean && vite build",
     "test": "vitest run",
+    "test:watch": "vitest --watch",
```

Comment:
If I make this change, the tests never stop running 😅 I'll remove the changes I added to the `package.json` since they have nothing to do with this PR. I just added them because it was helpful while doing the tests. I'll look into why the tests never stop running afterwards.